doc/plugins/redirect.md

```diff
@@ -0,0 +1,78 @@
+[中文](redirect.md)
+
+# redirect
+
+upstream proxy info rewrite plugin.
```

Comment:
need Chinese version.
and `upstream proxy info rewrite plugin` is confused, is it copy-paste?

Reply:
I'll create a todo issue about the Chinese version.